src/include/sof/drivers/ssp.h

```diff
@@ -156,6 +156,12 @@ extern const struct dai_driver ssp_driver;
 #define SSP_REG_MAX	SSIOC
 #endif
 
+/* SSTSA bits */
+#define SSTSA_TXEN		BIT(8)
+
+/* SSRSA bits */
+#define SSRSA_RXEN		BIT(8)
+
```

Comment:
nick-pick: the code doesn't use BIT for SSP, so either there's a follow-up to convert to the BIT usage or we use the existing style.

Reply:
ssp.h is messy in terms of BIT vs (1 << x), there are usage of both.

Reply:
I'm in favor of doing a conversion to BIT macro.

Reply:
Added usage of BIT in whole ssp.h file